ibridges/meta.py

```diff
@@ -59,11 +59,13 @@ def __iter__(self) -> Iterator:
         if self.blacklist is None:
             yield from self.item.metadata.items()
```

Comment:
There just needs to be a `return` after the yield and the code will work. Now you're iterating over the list twice.